specification/types.md

```diff
@@ -0,0 +1,37 @@
+# Types and Data Structures
+
+## Overview
+
+This document outlines some of the common types and data structures defined by OpenFeature and referenced elsewhere in this specification.
+
+### boolean
+
+A logical true or false, as represented idiomatically in the implementation languages.
+
+### string
+
+A UTF-8 encoded string.
+
+### number
+
+A numeric value of unspecified type or size. Implementation languages may further differentiate between integers, floating point numbers, and other specific numeric types and provide functionality as idioms dictate.
+
+### structure
+
+Structured data, presented however is idiomatic in the implementation language, such as JSON or YAML.
+
```

Comment:
`evaluation context` is missing from this section
```suggestion
### evaluation context

Structured data, presented however is idiomatic in the implementation language, such as JSON or YAML.

```

Reply:
This is not just the `evaluation context`. "Structure" is a very generic, language neutral way to say arbitrarily structured data. For example, the thing returned from the `getObjectValue` flag call.